src/ansys/fluent/core/utils/search.py

```diff
@@ -500,8 +500,17 @@ def _search_whole_word(
 
 def _download_nltk_data():
     """Download NLTK data on demand."""
+    import ssl
+
     import nltk
 
+    try:
+        _create_unverified_context = ssl._create_unverified_context
+    except AttributeError:
+        pass
+    else:
+        ssl._create_default_https_context = _create_unverified_context
```

Comment:
Minor thing: curious about the choice to use `else`, when does the exception happen? Is this code the same as:
```suggestion
    try:
        _create_unverified_context = ssl._create_unverified_context
        ssl._create_default_https_context = _create_unverified_context
    except AttributeError:
        pass
```
?

Reply:
```
import ssl

try:
    _create_unverified_https_context = ssl._create_unverified_context
except AttributeError:
    # Legacy Python that doesn't verify HTTPS certificates by default
    pass
else:
    # Handle target environment that doesn't support HTTPS verification
    ssl._create_default_https_context = _create_unverified_https_context
```